step/step_test.go

```diff
@@ -0,0 +1,143 @@
+package step
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/bitrise-io/go-xcode/xcodeproject/serialized"
+	"github.com/bitrise-io/go-xcode/xcodeproject/xcscheme"
+	"github.com/bitrise-steplib/steps-xcode-build-for-test/mocks"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GivenIosProjectProducesOneXctestrun_WhenFindTestBundle_ThenReturnsTestBundle(t *testing.T) {
```

Comment:
I do not have good alternative yet on how I would have done it differently and if you cannot provide an alternative yourself then do not comment on it say the wise people. So it is good as it is. But still want to share what I was thinking while looking at this. 😄 

It feels like it might be hard to write tests from the main `findTestBundle` entry point. As far as I can see these are the main logic tree:
```
- findTestBundle
   - findBuiltXCTestrunFiles
      - Get xctestrun paths
      - Filter them based on the modified time
      - Determine the default one based on the path
   - findBuiltTestDirPath
```
The inner ones could have been extracted into their own function / struct to focus on a single task and then the unit tests could be easier to write.